### PR TITLE
Update Russian localization

### PR DESCRIPTION
**Pull request by ghost**
Update Russian localization

### CODE REVIEW

File: res/values-ru/syberia_strings.xml [hunk 1]
```diff
@@ -13,6 +13,108 @@
 -->
 <resources xmlns:xliff="urn:oasis:names:tc:xliff:document:1.2">
 
+    <!-- App Ops Categories -->
+    <string name="app_ops_cat_location">Местоположение</string>
+    <string name="app_ops_cat_personal">Personal</string>
+    <string name="app_ops_cat_messaging">Messaging</string>
+    <string name="app_ops_cat_media">Media</string>
+    <string name="app_ops_cat_device">Устройство</string>
```
Comment (alexxxdev) [inline res/values-ru/syberia_strings.xml]: we will not translate these lines? @blinoff82 @burdanox

File: res/values-ru/syberia_strings.xml [hunk 2]
```diff
@@ -13,6 +13,108 @@
...
+    <string name="app_ops_manage_ipsec_tunnels">Управлять IP-туннелями</string>
+    <string name="app_ops_start_foreground">start foreground</string>
+    <string name="app_ops_bluetooth_scan">Сканирование блютуз</string>
+    <string name="app_ops_boot_completed">Автостарт</string>
+
+    <!-- App ops detail -->
+    <string name="app_ops_entry_summary"><xliff:g id="op">%1$s</xliff:g> (used <xliff:g id="count">%2$s</xliff:g>)</string>
+    <string name="app_ops_allowed_count">Разрешено <xliff:g id="count" example="2 times">%s</xliff:g></string>
+    <string name="app_ops_ignored_count">Отказано <xliff:g id="count" example="2 times">%s</xliff:g></string>
+    <string name="app_ops_both_count">Разрешено <xliff:g id="count">%1$s</xliff:g>, denied <xliff:g id="count">%2$s</xliff:g></string>
```
Comment (alexxxdev) [inline res/values-ru/syberia_strings.xml:105]: denied - we will not translate this word? @blinoff82 @burdanox

File: res/values-ru/syberia_strings.xml [hunk 3]
```diff
@@ -13,6 +13,108 @@
...
+    <string name="app_ops_accessibility_volume">accessibility volume</string>
+    <string name="app_ops_read_phone_numbers">Читать телефонные номера</string>
+    <string name="app_ops_request_install_packages">Устанавливать приложения</string>
+    <string name="app_ops_picture_in_picture">pip</string>
+    <string name="app_ops_app_start_foreground">start foreground</string>
+    <string name="app_ops_answer_phone_calls">Отвечать на звонки</string>
+    <string name="app_ops_run_any_in_background">Запуск в фоне</string>
+    <string name="app_ops_change_wifi_state">Изменять состояния WIFI</string>
+    <string name="app_ops_request_delete_packages">Удалять прилежения</string>
+    <string name="app_ops_bind_accessibility_service">bind to accessibility</string>
```
Comment (alexxxdev) [inline res/values-ru/syberia_strings.xml]: we will not translate it?

File: res/values-ru/syberia_strings.xml [hunk 4]
```diff
@@ -13,6 +13,108 @@
...
+    <string name="app_ops_read_external_storage">Читать внешнее хранилище</string>
+    <string name="app_ops_write_external_storage">Запись на внешнее хранилище</string>
+    <string name="app_ops_turn_on_screen">Включать экран</string>
+    <string name="app_ops_get_accounts">device accounts</string>
+    <string name="app_ops_run_in_background">Запуск в фоне</string>
+    <string name="app_ops_accessibility_volume">accessibility volume</string>
+    <string name="app_ops_read_phone_numbers">Читать телефонные номера</string>
+    <string name="app_ops_request_install_packages">Устанавливать приложения</string>
+    <string name="app_ops_picture_in_picture">pip</string>
+    <string name="app_ops_app_start_foreground">start foreground</string>
```
Comment (alexxxdev) [inline res/values-ru/syberia_strings.xml]: we will not translate it?

File: res/values-ru/syberia_strings.xml [hunk 5]
```diff
@@ -13,6 +13,108 @@
...
+    <string name="app_ops_assist_screenshot">Скриншот</string>
+    <string name="app_ops_use_body_sensors">body sensors</string>
+    <string name="app_ops_read_cell_broadcasts">cell broadcasts</string>
+    <string name="app_ops_mock_location">Фиктивное местоположение</string>
+    <string name="app_ops_read_external_storage">Читать внешнее хранилище</string>
+    <string name="app_ops_write_external_storage">Запись на внешнее хранилище</string>
+    <string name="app_ops_turn_on_screen">Включать экран</string>
+    <string name="app_ops_get_accounts">device accounts</string>
+    <string name="app_ops_run_in_background">Запуск в фоне</string>
+    <string name="app_ops_accessibility_volume">accessibility volume</string>
```
Comment (alexxxdev) [inline res/values-ru/syberia_strings.xml]: we will not translate it?

File: res/values-ru/syberia_strings.xml [hunk 6]
```diff
@@ -13,6 +13,108 @@
...
+    <string name="app_ops_change_wallpaper">Изменять обои</string>
+    <string name="app_ops_assist_structure">assist structure</string>
+    <string name="app_ops_assist_screenshot">Скриншот</string>
+    <string name="app_ops_use_body_sensors">body sensors</string>
+    <string name="app_ops_read_cell_broadcasts">cell broadcasts</string>
+    <string name="app_ops_mock_location">Фиктивное местоположение</string>
+    <string name="app_ops_read_external_storage">Читать внешнее хранилище</string>
+    <string name="app_ops_write_external_storage">Запись на внешнее хранилище</string>
+    <string name="app_ops_turn_on_screen">Включать экран</string>
+    <string name="app_ops_get_accounts">device accounts</string>
```
Comment (alexxxdev) [inline res/values-ru/syberia_strings.xml]: we will not translate it?

File: res/values-ru/syberia_strings.xml [hunk 7]
```diff
@@ -13,6 +13,108 @@
...
+    <string name="app_ops_picture_in_picture">pip</string>
+    <string name="app_ops_app_start_foreground">start foreground</string>
+    <string name="app_ops_answer_phone_calls">Отвечать на звонки</string>
+    <string name="app_ops_run_any_in_background">Запуск в фоне</string>
+    <string name="app_ops_change_wifi_state">Изменять состояния WIFI</string>
+    <string name="app_ops_request_delete_packages">Удалять прилежения</string>
+    <string name="app_ops_bind_accessibility_service">bind to accessibility</string>
+    <string name="app_ops_accept_handover">accept handover</string>
+    <string name="app_ops_manage_ipsec_tunnels">Управлять IP-туннелями</string>
+    <string name="app_ops_start_foreground">start foreground</string>
```
Comment (alexxxdev) [inline res/values-ru/syberia_strings.xml]: we will not translate it?

File: res/values-ru/syberia_strings.xml [hunk 8]
```diff
@@ -13,6 +13,108 @@
...
+    <string name="app_ops_write_sms">Писать SMSe</string>
+    <string name="app_ops_use_fingerprint">Датчик отпечатка пальцев</string>
+    <string name="app_ops_add_voicemail">Голосовая почта</string>
+    <string name="app_ops_read_phone_state">access phone state</string>
+    <string name="app_ops_scan_wifi">Сканирование WiFi сетей</string>
+    <string name="app_ops_change_wallpaper">Изменять обои</string>
+    <string name="app_ops_assist_structure">assist structure</string>
+    <string name="app_ops_assist_screenshot">Скриншот</string>
+    <string name="app_ops_use_body_sensors">body sensors</string>
+    <string name="app_ops_read_cell_broadcasts">cell broadcasts</string>
```
Comment (alexxxdev) [inline res/values-ru/syberia_strings.xml]: we will not translate it?

File: res/values-ru/syberia_strings.xml [hunk 9]
```diff
@@ -13,6 +13,108 @@
...
+    <string name="app_ops_use_media_volume">Громкость медиа</string>
+    <string name="app_ops_use_notification_volume">Громкость уведомлений</string>
+    <string name="app_ops_use_ring_volume">Громкость рингтона</string>
+    <string name="app_ops_use_vibrate">Виброотклик</string>
+    <string name="app_ops_use_voice_volume">voice call volume</string>
+    <string name="app_ops_write_mms">Писать MMS</string>
+    <string name="app_ops_write_sms">Писать SMSe</string>
+    <string name="app_ops_use_fingerprint">Датчик отпечатка пальцев</string>
+    <string name="app_ops_add_voicemail">Голосовая почта</string>
+    <string name="app_ops_read_phone_state">access phone state</string>
```
Comment (alexxxdev) [inline res/values-ru/syberia_strings.xml]: we will not translate it?

File: res/values-ru/syberia_strings.xml [hunk 10]
```diff
@@ -13,6 +13,108 @@
...
+    <string name="app_ops_use_alarm_volume">Громкость будильника</string>
+    <string name="app_ops_use_audio_focus">audio focus</string>
+    <string name="app_ops_use_bluetooth_volume">Громкость блютуз</string>
+    <string name="app_ops_use_master_volume">Общая громкость</string>
+    <string name="app_ops_use_media_buttons">Медиа-кнопки</string>
+    <string name="app_ops_use_media_volume">Громкость медиа</string>
+    <string name="app_ops_use_notification_volume">Громкость уведомлений</string>
+    <string name="app_ops_use_ring_volume">Громкость рингтона</string>
+    <string name="app_ops_use_vibrate">Виброотклик</string>
+    <string name="app_ops_use_voice_volume">voice call volume</string>
```
Comment (alexxxdev) [inline res/values-ru/syberia_strings.xml]: we will not translate it?

**Review by alexxxdev — CHANGES_REQUESTED** [review]

**Review by ghost — COMMENTED** [review]
solved

**Review by alexxxdev — APPROVED** [review]

**Pull request merged**